File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/d/Cidiagdto.java
package iih.ci.mrqc.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class Cidiagdto extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键标识
	 * @return String
	 */
	public String getId_cididt() {
		return ((String) getAttrVal("Id_cididt"));
	}
	/**
	 * 主键标识
	 * @param Id_cididt
	 */
	public void setId_cididt(String Id_cididt) {
		setAttrVal("Id_cididt", Id_cididt);
	}
	/**
	 * 诊断类型
	 * @return String
	 */
	public String getId_disys() {
		return ((String) getAttrVal("Id_disys"));
	}
	/**
	 * 诊断类型
	 * @param Id_disys
	 */
	public void setId_disys(String Id_disys) {
		setAttrVal("Id_disys", Id_disys);
	}
	/**
	 * 诊断名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 诊断名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 诊断编码
	 * @return String
	 */
	public String getCode() {
		return ((String) getAttrVal("Code"));
	}
	/**
	 * 诊断编码
	 * @param Code
	 */
	public void setCode(String Code) {
		setAttrVal("Code", Code);
	}
	/**
	 * 诊断描述
	 * @return String
	 */
	public String getDes() {
		return ((String) getAttrVal("Des"));
	}
	/**
	 * 诊断描述
	 * @param Des
	 */
	public void setDes(String Des) {
		setAttrVal("Des", Des);
	}
	/**
	 * 诊断日期
	 * @return String
	 */
	public String getDt_di() {
		return ((String) getAttrVal("Dt_di"));
	}
	/**
	 * 诊断日期
	 * @param Dt_di
	 */
	public void setDt_di(String Dt_di) {
		setAttrVal("Dt_di", Dt_di);
	}
	/**
	 * 就诊
	 * @return String
	 */
	public String getId_en() {
		return ((String) getAttrVal("Id_en"));
	}
	/**
	 * 就诊
	 * @param Id_en
	 */
	public void setId_en(String Id_en) {
		setAttrVal("Id_en", Id_en);
	}
	/**
	 * 患者
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 诊断过程
	 * @return String
	 */
	public String getId_ditp() {
		return ((String) getAttrVal("Id_ditp"));
	}
	/**
	 * 诊断过程
	 * @param Id_ditp
	 */
	public void setId_ditp(String Id_ditp) {
		setAttrVal("Id_ditp", Id_ditp);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/cidiagdto/d/Cidiagdto.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mrqc.d
{
    /// <summary>
    /// Cidiagdto 的摘要说明。
    /// </summary>
    public class Cidiagdto : BaseDTO {

        public Cidiagdto() {
 
        }

        /// <summary>
        /// 主键标识
        /// </summary>
		public string Id_cididt {
            get { return getAttrVal<string>("Id_cididt",null); }
            set { setAttrVal<string>("Id_cididt", value); }
        }

        /// <summary>
        /// 诊断类型
        /// </summary>
		public string Id_disys {
            get { return getAttrVal<string>("Id_disys",null); }
            set { setAttrVal<string>("Id_disys", value); }
        }

        /// <summary>
        /// 诊断名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 诊断编码
        /// </summary>
		public string Code {
            get { return getAttrVal<string>("Code",null); }
            set { setAttrVal<string>("Code", value); }
        }

        /// <summary>
        /// 诊断描述
        /// </summary>
		public string Des {
            get { return getAttrVal<string>("Des",null); }
            set { setAttrVal<string>("Des", value); }
        }

        /// <summary>
        /// 诊断日期
        /// </summary>
		public string Dt_di {
            get { return getAttrVal<string>("Dt_di",null); }
            set { setAttrVal<string>("Dt_di", value); }
        }

        /// <summary>
        /// 就诊
        /// </summary>
		public string Id_en {
            get { return getAttrVal<string>("Id_en",null); }
            set { setAttrVal<string>("Id_en", value); }
        }

        /// <summary>
        /// 患者
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 诊断过程
        /// </summary>
		public string Id_ditp {
            get { return getAttrVal<string>("Id_ditp",null); }
            set { setAttrVal<string>("Id_ditp", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_cididt", "Id_disys", "Name", "Code", "Des", "Dt_di", "Id_en", "Id_pat", "Id_ditp"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mrqc.d.Cidiagdto";
        }
    }
}
